astropy/convolution/kernels.py

```diff
@@ -697,7 +697,7 @@ class AiryDisk2DKernel(Kernel2D):
     2D Airy disk kernel.
 
     This kernel models the diffraction pattern of a circular aperture. This
-    kernel is normalized to a peak value of 1.
+    kernel is normalized so that it sums to 1.
```

Comment:
```suggestion
    kernel is normalized so that it integrates to 1.
```
Seems better for the mathematical property, even if it amounts to a numerical summation. And it is achieved by calling the `self.normalize([mode='integral'])` method.